.circleci/config.yml

```diff
@@ -22,12 +31,21 @@ commands:
       - run: make build
       - run: make release
 
-  common_setup:
-    description: Prepare the repository for work
+  perform_opera_nightly_integration_testing:
+    description: Test xOpera CLI commands with testing service template
     steps:
       - checkout
-      - run: sudo pip install pipenv
-      - run: make init
+      - run: pip install --user --upgrade pip
+      - run: pip install --user wheel
+      - run: pip install --user opera
+      - run:
+          command: |
+            cd tests/integration
+            for integration_test in *; do
+              cd "${integration_test}"
+              sh runme.sh opera
+              cd ..
+            done
```

Comment:
We should be able to reuse the make definition for this. See Makefile's suggestions below.

Reply:
Ok, I will use `make integration_test` here. I assume that I will need to export `OPERA` env var here to use the opera installed from PyPI and not the one from pipenv. Is that right?

Reply:
Exactly. Just provide `OPERA` environment variable via some CircleCI means and we should be gold.